hermeto/core/package_managers/general.py

```
@@ -157,38 +155,22 @@ async def _async_download_binary_file(
 
+    try:
         async with session.get(
-            url,
-            timeout=timeout,
-            raise_for_status=True,
-            ssl=ssl_context,
-            headers=headers,
-        ) as resp:
+            url, timeout=timeout, raise_for_status=True, ssl=ssl_context, headers=headers
+        ) as response:
             with open(download_path, "wb") as f:
```

Ruff (0.15.21)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)